src/app/(homepage)/page.tsx

```diff
@@ -138,7 +147,11 @@ export default function Home() {
           title="News & Blog"
           description="The latest updates and announcements from the Filecoin ecosystem and Filecoin Foundation."
         >
-          <FeaturedBlogPosts />
+          {hasFeaturedBlogPosts ? (
```

Comment:
We probably shouldn't show the entire section if there are no featuredBlogPosts..

Reply:
Same comment [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/951/files#r1896524926)